gha-webhook-listener.py

```diff
@@ -251,8 +253,14 @@ def deploy_tarball(artifact_url: str, target_dir: str) -> None:
                 raise
 
     logger.info("...download complete.")
+    if arg_symlink:
+        create_symlink(source=target_dir, linkname=arg_symlink)
 
-    create_symlink(source=target_dir, linkname=arg_symlink)
+    if arg_hook_script is not None:
+        logger.info("...running hook script")
```

Comment:
The ... in "... download complete" is meant to match the "..." in "Fetching artifact ...". No need for another here. Also I think it will be helpful to log the name of the script.

```suggestion
        logger.info("Running hook script '%s'", arg_hook_script)
```
```suggestion
        logger.info("Running hook script %s", )
```